bot/code_review_bot/__init__.py

```diff
@@ -68,7 +68,7 @@ def __init__(
         self.path = path
         self.line = line
         self.nb_lines = nb_lines
-        self.check = check
+        self.check = check or "unknown"
```

Comment:
This basically happens for analyzers which only have one rule, and so they don't specify it.
Instead of "unknown", maybe it'd be better to do one of:
- Use None;
- Use "default";
- Use the analyzer name itself, since that's basically the rule.

Another thing: should we do it here or for MozLint only? I'm not sure.

Reply:
You're right, it makes more sense to only do it in Mozlint. The issue arises because Mozlint supports many different analyzers & their settings